pkg/controller/deployment/util.go

```diff
@@ -0,0 +1,11 @@
+package deployment
```

Comment:
there shall be no `util.go`. Move to place of use.

Reply:
Done